sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/encoders/ExpressionEncoder.scala

```diff
@@ -87,7 +87,9 @@ object ExpressionEncoder {
       }
       constructProjection(row).get(0, anyObjectType).asInstanceOf[T]
     } catch {
-      case e: SparkRuntimeException if e.getCondition == "NOT_NULL_ASSERT_VIOLATION" =>
+      case e: SparkRuntimeException if
+        e.getCondition == "NOT_NULL_ASSERT_VIOLATION" ||
```

Comment:
nit: If I am not mistaken, from style perspective we usually leave conditions up to 100 characters and then break.